plugin/trino-kudu/src/test/java/io/trino/plugin/kudu/TestKuduConnectorTest.java

```diff
@@ -716,6 +716,11 @@ public void testLimitPushdown()
         assertThat(query("SELECT name FROM nation LIMIT 30")).isNotFullyPushedDown(LimitNode.class); // Use high limit for result determinism
     }
 
+    // Empty override is to disable test due to flakiness per https://nineinchnick.github.io/trino-cicd/reports/flaky/
```

Comment:
Please leave GitHub issue link instead of https://nineinchnick.github.io/trino-cicd/reports/flaky/.